EasyPost.Tests/ServicesTests/EventServiceTest.cs

```diff
@@ -54,6 +67,65 @@ public async Task TestRetrieve()
             Assert.Equal(@event.Id, retrievedEvent.Id);
         }
 
+        [Fact]
+        [CrudOperations.Read]
+        [Testing.Function]
+        public async Task TestRetrievePayloadsForEvent()
+        {
+            UseVCR("retrieve_payloads_for_event");
+
+            // Create a webhook to receive the event
+            const string url = "https://example.com";
+            Webhook webhook = await Client.Webhook.Create(new Dictionary<string, object> { { "url", url } });
+            CleanUpAfterTest(webhook.Id);
+
+            // Create a batch to trigger an event
+            Batch _ = await Client.Batch.Create(new Dictionary<string, object> { { "shipments", new List<Dictionary<string, object>> { Fixtures.BasicShipment } } });
+
+            // Wait for the event to be created
+            await Task.Delay(5000);
```

Comment:
Please wrap this in the same logic we use elsewhere to ensure it only happens on record.